spec/controllers/my/all_controller_spec.rb

```diff
@@ -43,34 +43,6 @@
     Collection.destroy_all
   end
 
-  describe "#index" do
- 
-    describe "User" do
-      before do
-        sign_in user
-      end
-
-      it "should respond with success" do
-        get :index, use_route: :dashboard
-        expect(response).to be_successful
-      end
-
-      it "shows the correct files" do
-        get :index, use_route: :dashboard
-        # shows documents I deposited
-        expect(assigns[:document_list].map(&:id)).to include(@my_file.id)
-        # doesn't show collections
-        expect(assigns[:document_list].map(&:id)).to_not include(@my_collection.id)
-        # doesn't show shared files
-        expect(assigns[:document_list].map(&:id)).to include(@shared_file.id)
-        # doesn't show other users' files
-        expect(assigns[:document_list].map(&:id)).to_not include(@unrelated_file.id)
-        # doesn't show non-generic files
-        expect(assigns[:document_list].map(&:id)).to_not include(@wrong_type.id)
-      end
-    end
-  end
```

Comment:
First thought is why did you remove the old test? 

Second thought is don't we want to verify that non-admin users get an unsuccessful response?